tools/buildsys/src/args.rs

```diff
@@ -134,6 +134,18 @@ pub(crate) struct BuildVariantArgs {
     #[arg(long, env = "BUILDSYS_VARIANT")]
     pub(crate) variant: String,
 
+    #[arg(long, env = "BUILDSYS_VARIANT_PLATFORM")]
+    pub(crate) variant_platform: String,
+
+    #[arg(long, env = "BUILDSYS_VARIANT_RUNTIME")]
+    pub(crate) variant_runtime: String,
+
+    #[arg(long, env = "BUILDSYS_VARIANT_FAMILY")]
+    pub(crate) variant_family: String,
+
+    #[arg(long, env = "BUILDSYS_VARIANT_FLAVOR")]
+    pub(crate) variant_flavor: String,
```

Comment:
Here I opted to add these to `BuildVariantArgs`, even though they're also present in `BuildPackageArgs`, because my goal is to eventually eliminate them from `BuildPackageArgs` when conditional package builds are eliminated.